lib/ember_cli/route_helpers.rb

```diff
@@ -1,8 +1,14 @@
 require "ember_cli/html_constraint"
+require "ember_cli/rails_info_constraint"
 
 module ActionDispatch
   module Routing
     class Mapper
+      MULTI_CONSTRAINT = -> (req) {
```

Comment:
Really? I'm personally a fan of the idea @jimweirich put forward about blocks (paraphrased):

> Curly brackets are for functional blocks, where the primary purpose of the block is to return a value. `do ... end` is for procedural blocks, where the primary purpose of the block is to have a side-effect.

That said, I'm happy to comply with your style guide if it pleases the bots.